src/main/java/ch/naviqore/gtfs/schedule/GtfsScheduleParser.java

```diff
@@ -121,4 +117,25 @@ private void parseTransfers(CSVRecord record) {
                 TransferType.parse(record.get("transfer_type")),
                 minTransferTime.isEmpty() ? null : Integer.parseInt(record.get("min_transfer_time")));
     }
+
+    private static class ParsingHelpers {
+
+        private static String getStringField(CSVRecord record, String fieldName) {
```

Comment:
This method seems superfluous from my point of view. I would inline `getStringField` and `getIntField` since they are not used anywhere outside this inner class.

Reply:
6a132d7